_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,248 @@
+---
+id: bc4a75
+name: ARIA required owned elements
+rule_type: atomic
+description: |
+  This rule checks that each role has at least one of its required owned elements.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Audrey Maniez
+  - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has a [semantic role](#semantic-role) that has [WAI-ARIA required owned elements](https://www.w3.org/TR/wai-aria/#mustContain).
+
+## Expectation
+
+The target element either [owns](#owned-by) at least one instance of one of its [WAI-ARIA required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) or has `aria-busy` attribute set to `true`.
+
+## Assumptions
+
+Attributes that start with `aria-*` are used to set either [state](https://www.w3.org/WAI/PF/aria-1.1/terms#def_state) or [property](https://www.w3.org/WAI/PF/aria-1.1/terms#def_property) that should be picked up by assistive technologies. If any `aria-*` attributes are not used for that purpose, the results of this rule may be incorrect.
+
+## Accessibility Support
+
+This rule relies on assistive technologies to recognize owned elements, as defined by [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria). This includes when they are nested descendants that are not immediate children. However, some assistive technologies do not recognize owned elements that are not immediate children, unless workarounds are used.
+
+## Background
+
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+- [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
+- [HTML in ARIA](https://www.w3.org/TR/html-aria/)
+- [Implicit WAI-ARIA Semantics](https://www.w3.org/TR/wai-aria-1.1/#implicit_semantics)
+
+## Test Cases
+
+### Passed
```

Comment:
Some suggestions for further pass examples
- One passed example with aria-busy
- One passed example where the top-level element in the example does not have an explicit role (since it is no longer an applicability requirement)
- One passed example where there are elements with roles that meet the owned by requirements, but also other elements that do not

Reply:
Added examples for points 1 & 2, as for 3 believe passed example 1 and 4 convey this already.